Lib/fontbakery/profiles/adobefonts.py

```diff
@@ -1,6 +1,8 @@
 """
 Checks for Adobe Fonts (formerly known as Typekit).
 """
+import unicodedata
```

Comment:
do we have this dependency on requirements.txt and setup.py?

Reply:
It's a module that's built into Python.

Reply:
ah, ok then :-)